src/main/java/duke/Event.java

```diff
@@ -0,0 +1,41 @@
+package main.java.duke;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
```

Comment:
Separation of package statements as per [guidelines](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements)
```suggestion
package main.java.duke;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;
```